saleor/core/taxes.py

```diff
@@ -46,3 +46,12 @@ class TaxData:
     shipping_price_net_amount: Decimal
     shipping_tax_rate: Decimal
     lines: list[TaxLineData]
+
+
+class TaxDataErrorMessage:
+    EMPTY = "Empty tax data."
+    NEGATIVE_VALUE = "Tax data contains negative values."
+    LINE_NUMBER = (
+        "Number of lines from tax data doesn't match the line number from order."
+    )
+    OVERFLOW = "Tax data contains overflow numbers."
```

Comment:
What do you mean exactly?

Reply:
Tax rates > 100% and prices > 999.999.999